mnist/README.md

```diff
@@ -251,7 +251,7 @@ spec:
        ...
        ```
 
-  1. Next we need to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` so that our code knows
+  2. Next we need to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` so that our code knows
```

Comment:
Why did you change the numbering? We should be letting markdown autonumber the items.

Reply:
I'm sorry, done by mistake. I will get latest updates from master and remove these changes.